src/database/config/config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
+
+module.exports.development = {
+  url: process.env.DATABASE_URL,
```

Comment:
Remember to add this connection string of the database in `.env` file, because .env file is in `.gitignore` and can't be uploaded to GitHub you will add the connection example in `.env.example`